src/main/java/com/synopsys/integration/alert/web/controller/HomeController.java

```diff
@@ -60,17 +64,23 @@ public String index() {
         final HttpServletRequest httpRequest = request;
         final CsrfToken csrfToken = csrfTokenRespository.loadToken(request);
         final Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-        final boolean hasAlertRole = authorizationManager.hasAlertRole();
-        final boolean authorized = authentication.isAuthenticated() && hasAlertRole && csrfToken != null;
+        final boolean isAnonymous = authentication.getAuthorities().stream()
+                                        .map(authority -> authority.getAuthority())
+                                        .collect(Collectors.toSet())
+                                        .contains("ROLE_ANONYMOUS");
+        final boolean authorized = authentication.isAuthenticated() && !isAnonymous && csrfToken != null;
 
         if (!authorized) {
             httpRequest.getSession().invalidate();
             return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
         } else {
-            final String body = "{\"message\":\"Authenticated\"}";
+            final ResponseBodyBuilder responseBody = new ResponseBodyBuilder("Authenticated");
+            responseBody.put("saml_enabled", samlContext.isSAMLEnabled());
+
             final HttpHeaders headers = new HttpHeaders();
             headers.add(csrfToken.getHeaderName(), csrfToken.getToken());
-            return new ResponseEntity<>(body, headers, HttpStatus.OK);
+
+            return responseFactory.createResponse(HttpStatus.OK, headers, responseBody.build());
```

Comment:
Maybe we could return a 204 or something here rather than driving our UI with the `"Authenticated"` String from above?